symphony-bdk-core/src/main/java/com/symphony/bdk/core/client/ApiClientFactory.java

```diff
@@ -69,6 +70,21 @@ public ApiClient getPodClient() {
     return buildClient(POD_CONTEXT_PATH, this.config.getPod());
   }
 
+
+  /**
+   * Returns a fully initialized {@link ApiClient} for Pod API.
+   * If common jwt is enabled, the client is going to set the Authorization header, removing the sessionToken.
+   *
+   * @return a new {@link ApiClient} instance.
+   */
+  public ApiClient getPodClient(AuthSession authSession) {
```

Comment:
I would suggest having two methods: one with non-null `authSession`, another with no parameters.

Reply:
we have two functions getPodClient(authSession) and the already existing getPodClient(). I forgot to use it here though, I made the change.